frontend/src/components/VBackToSearchResultsLink.vue

```diff
@@ -3,14 +3,18 @@
   <VLink
     class="time inline-flex flex-row items-center gap-2 rounded-sm p-2 text-xs font-semibold text-dark-charcoal-70 pe-3 hover:text-dark-charcoal"
     v-bind="$attrs"
+    @click="handleClick"
```

Comment:
I checked out the PR locally and it seems like this handler is not being called, despite being perfectly ok from a code standpoint. In fact, the identical code from a different component `VHomeGallery.vue` works just as expected.

https://github.com/WordPress/openverse/blob/5106dc3e73d0b55f8eb0bfb8375c103c93402c1a/frontend/src/components/VHomeGallery/VHomeGallery.vue#L32

Reply:
Yes Dhruv, the `CLICK_HOME_GALLERY_IMAGE` gets fired as expected, but `BACK_TO_SEARCH` event doesn't

Reply:
We'll try to investigate, but in the meantime please feel free to work on another issue if you'd like.

Reply:
> We'll try to investigate, but in the meantime please feel free to work on another issue if you'd like.

Sure, thanks!